### ISSUE TITLE
Tags and templates should be used when extension is created as well as on run

### ISSUE THREAD

**Issue by hayleyavw** [uccser/kordac#44]
Current use:
```
converter = kordac.Kordac()
result = converter.run(s, tags=[], templates={})
```
Instead the tags and templates should be specified when the Kordac instance is created, with the option to update these tags/templates (and eventually regex) between runs.

**Comment by JackMorganNZ** [comment]
Do we want the ability to provide new regexes? Were you thinking of new extensions instead?

If you changed a regex, our docs would be pointless for that tag. Plus if you were adding new parameters, you would need to change the Python code as well.

**Comment by hayleyavw** [comment]
renamed because can't remember why we included regex

**Comment by JackMorganNZ** [comment]
Fixed in 54ad3b14a89339a55cad15f449c9aa1841011802
